src/pages/Homepage/components/CustomersFeedback.jsx

```diff
@@ -0,0 +1,101 @@
+import {
+  Box,
+  Container,
+  Flex,
+  Heading,
+  Text,
+  Image
+} from '@chakra-ui/react';
+import { ReactElement } from 'react';
+
+import customer_1 from '../../../images/customer-1.png';
+import customer_2 from '../../../images/customer-2.png';
+import customer_3 from '../../../images/customer-3.png';
+
+const Card = ({ name, description, logo, date }) => {
+  return (
+    <Box
+      maxW={"full"}
+      w={'full'}
+      borderWidth="1px"
+      borderRadius="20px"
+      borderColor="#ECECEC"
+      overflow="hidden"
+      p={{ base: '32px 24px 24px', md: '32px 24px 24px' }}>
+      <Box align={{ base: 'left', md: 'center' }}>
+        <Image src={logo} alt="" w={"100%"}  maxW={{ base: '120px', md: '122px' }} />
+        <Box mt={{ base: '12px', md: '24px' }}>
+          <Text
+            maxW={"800px"}
+            mt={{ base: '48px', md: '48px' }}
+            fontSize={{ base: '14px', md: '16px' }}
+            lineHeight={'140%'}
+            fontWeight={500}>
+            {description}
+          </Text>
+          <Box mt={"58px"}>
+            <Box
+              fontWeight={"500"}
+              fontSize={"20px"}
+              lineHeight={"124%"}
+              color={"#3722DA"}
+              textTransform={"uppercase"}
+            >
+              {name}
+            </Box>
+            <Box
+              mt={"8px"}
+              fontWeight={"500"}
+              fontSize={"14px"}
+              lineHeight={"140%"}
+              color={"#6C6C6C"}>
+              {date}
+            </Box>
+          </Box>
+        </Box>
+      </Box>
+    </Box>
+  );
+};
+
+const CustomersFeedback = () => {
+  return (
+    <Box w={"100%"}>
+
+      <Container w={"100%"} maxW={'1278px'} py={0} px={{ base: "24px", xl: 0 }}>
+        <Heading textAlign={"center"} fontSize={{ base: '24px', md: '36px' }} fontWeight={'500'} lineHeight={"131%"}>
+          What Our Customers Think About Us
+        </Heading>
+
+        <Flex mt={{ base: '24px', md: '40px' }} gridGap={{ base: '17px', md: '24px' }} justify="center" direction={{ base: 'column', xl: 'row' }}>
+          <Card
```

Comment:
You can create array with cards data, and simply map through it
This will reduce amount of code you are writting